src/store/rootReducer.ts

```diff
@@ -1,8 +1,10 @@
 import { combineReducers } from 'redux';
+import directionReducer from '../modules/direction/store/directionSlice';
 import mainReducer from '../modules/main/store/mainSlice';
 
 const rootReducer = combineReducers({
   main: mainReducer,
+  direction: directionReducer,
```

Comment:
we need following structure for direction: `{ [%DirectionsEnumKey%]: directionReducer }`, so the overall structure will be: 
```
{
  main: MainReducer,
  directions: {
    covid19: directionReducer,
    therapy: directionReducer,
    // ...
  }
```
}